tests/test_elements.py

```diff
@@ -28,20 +24,15 @@ def test_ThickElement():
     # Try to assign negative length and
     # detect validation error without breaking pytest
     element_length = -1.0
-    passed = True
-    try:
+    with pytest.raises(ValidationError):
         element.length = element_length
-    except ValidationError as e:
-        print(e)
-        passed = False
-    assert not passed
```

Comment:
Great to learn that this can be done in two lines - didn't know that.

Reply:
Yep! :) I use this in pyAMReX quite a bit to ensure we do not segfault but can recover on issues.